examples/psyad/lfric/adjoint_partial/.gitignore

```diff
@@ -0,0 +1,18 @@
+adj_calc_exner_pointwise_mod.F90
```

Comment:
I think this can just be `*.f90`?

Reply:
Unfortunately not, I tried and `git status` sees it. I refreshed the branch with the master and will send the PR back for review.

Reply:
Oops, I meant `*.F90`! Are you sure it doesn't work?

Reply:
All of them have `*.F90` extensions. I literally copied and pasted output from `git status` on the master after running examples 😄

Reply:
As per PSyclone Telco conversation this morning, I modifed the file to generic `*.F90`. Back to @arporter for another review.